Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_102.AdoPat.sarif

```diff
@@ -81,51 +81,6 @@
             "ValidationFingerprint/v2": "{\"secret\":\"h5lxeqkz4zujkyvzg5emkejgelxj4x64tvyzclprekuloawbbz5q\"}"
           },
           "rank": 62.14
-        },
-        {
-          "ruleId": "SEC101/102",
-          "ruleIndex": 0,
-          "level": "note",
-          "message": {
-            "id": "Default",
-            "arguments": [
-              "h5lxeq…",
-              "an apparent ",
-              "",
-              "Azure DevOps personal access token",
-              "",
-              ""
-            ]
-          },
-          "locations": [
-            {
-              "physicalLocation": {
-                "artifactLocation": {
```

Comment:
>t [](http://example.com/codeflow?start=29&length=1)

why did this go away? In general, it's good to explicitly account for results that disappear. 

We have either introduced new false negatives or you have potentially dropped false positives, right?

Where's the description, if so, in ReleaseHistory.md?

Reply:
would appear.
With my current change, we will ask for the static or dynamic validator class, removing duplications